qutebrowser/utils/log.py

```diff
@@ -291,6 +303,20 @@ def _init_formatters(level, color, force_color, json_logging):
     return console_formatter, ram_formatter, html_formatter, use_colorama
 
 
+def change_console_formatter(level):
+    """Change console formatter based on level.
+
+    Args:
+        level: The numeric logging level
+    """
+    if isinstance(console_handler.formatter, ColoredFormatter):
```

Comment:
Just to elaborate: When I looked at it, I was like "huh... why would `console_handler.formatter` ever not be a `ColoredFormatter`? Then after looking at the rest of `log.py` I remembered the JSON formatter. If that happens to me, it might happen to someone new to the code as well :wink: